Tools/BatchScripts/batch_summit.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 
-# Copyright 2019 Maxence Thevenet
+# Copyright 2019-2020 Maxence Thevenet, Axel Huebl
 #
 # This file is part of WarpX.
 #
 # License: BSD-3-Clause-LBNL
+#
+# Refs.:
+#   https://jsrunvisualizer.olcf.ornl.gov/?s4f0o11n6c7g1r11d1b1l0=
+#   https://docs.olcf.ornl.gov/systems/summit_user_guide.html#cuda-aware-mpi
 
 #BSUB -P <allocation ID>
 #BSUB -W 00:10
 #BSUB -nnodes 2
+#BSUB -alloc_flags smt4
 #BSUB -J WarpX
 #BSUB -o WarpXo.%J
 #BSUB -e WarpXe.%J
 
 module load gcc
 module load cuda
 
-omp=1
-export OMP_NUM_THREADS=${omp}
-
-num_nodes=$(( $(printf '%s\n' ${LSB_HOSTS} | sort -u | wc -l) - 1 ))
-jsrun -n ${num_nodes} -a 6 -g 6 -c 6 --bind=packed:${omp} <path/to/executable> <input file> > output.txt
+export OMP_NUM_THREADS=1
```

Comment:
Would it be useful to add a comment/NOTE/TODO that explains that we are setting `OMP_NUM_THREADS` because "(note: needs export OMP_NUM_THREADS=1 for the way we #ifdef OpenMP currently...)".
In case someone decides to ramp up OMP_NUM_THREADS without knowing what in the code will not work?